go.mod

```diff
@@ -65,7 +65,7 @@ require (
 	k8s.io/client-go v0.24.7
 	k8s.io/cloud-provider v0.24.7
 	k8s.io/component-base v0.24.7
-	k8s.io/cri-api v0.24.7
+	k8s.io/cri-api v0.25.0
```

Comment:
I wonder if that's a good idea... Maybe we need to skip the backport, because of dependency issues?

Reply:
OTOH, the version gets overridden anyways in the replace section at the end of the file: `k8s.io/cri-api => k8s.io/cri-api v0.24.7`